# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.h
# Commit: 93367fdd03ec
# Full Hash: 93367fdd03ec4b97b3a182f821b01fd53063196c
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Update threading invariant of AudioConduit::GetLocalSSRCs to match mRecvStreamConfig comment. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102279
# ==============================================================================

diff -r 9b2bc72201da -r 93367fdd03ec dom/media/webrtc/libwebrtcglue/AudioConduit.h
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.h	Fri Jan 29 15:37:11 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.h	Fri Jan 29 15:38:10 2021 +0100
@@ -273,13 +273,13 @@
   // main thread.
   const RefPtr<WebRtcCallWrapper> mCall;
 
-  // Written only on main thread. Guarded by mMutex, except for reads on main.
+  // Accessed only on main thread.
   webrtc::AudioReceiveStream::Config mRecvStreamConfig;
 
   // Written only on main thread. Guarded by mMutex, except for reads on main.
   webrtc::AudioReceiveStream* mRecvStream;
 
-  // Written only on main thread. Guarded by mMutex, except for reads on main.
+  // Accessed only on main thread.
   webrtc::AudioSendStream::Config mSendStreamConfig;
 
   // Written only on main thread. Guarded by mMutex, except for reads on main.
